# motion def:
# 静止　    _ceshi_rest
# 匀速  _ceshi_uniform
# 加减速    _ceshi_acc_dec
# 快速转弯  _ceshi_corner_fast
# all： _ceshi_all_motion
motion_file: _ceshi_all_motion.csv #
# IMU params:
imu: 1
# sample frequency of simulated GNSS/IMU data:
sample_frequency:
    imu: 100.0
    gps: 10.0
# topic name:
topic_name_imu: /sim/sensor/imu
topic_name_gt:  /pose/ground_truth
# output rosbag path:
output_path: /home/lxq/dataset/catkin_multi/shnelan-multi-sensor-fusion/Sensor_multi_fusion_DEBUG/workspace-ch6/src/9_data/ceshi
# output name:
output_name: _ceshi_all_motion.bag